app/controllers/collections_controller.rb

```diff
@@ -165,4 +165,22 @@ def destroy
     redirect_to(collections_url)
   end
 
+  private
+
+  def collection_params
+    params.require(:collection).permit(
+      :name, :title, :email, :header_image_url, :description,
```

Comment:
Ooh! Sorry, one more thing -- we need to add `:delete_icon` or else we can't delete an icon that's been added to a collection.